### PR TITLE
docs: correct english description at README

### PR DESCRIPTION
**Pull request by atian25**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,11 @@ Node.js >= 6.0.0 required.
 - [Plugins](https://github.com/search?q=topic%3Aegg-plugin&type=Repositories)
 - [Frameworks](https://github.com/search?q=topic%3Aegg-framework&type=Repositories)
 
-> English docs is still working in process, see [#360](https://github.com/eggjs/egg/issues/360).
+> All the source, comment and README is english.
```
Comment (shaoshuai0102) [inline README.md]: all the sources, comments and the README is in English

File: README.md [hunk 2]
```diff
@@ -43,7 +43,11 @@ Node.js >= 6.0.0 required.
...
+>
+> Only guide docs is still working in process, see [#363](https://github.com/eggjs/egg/issues/363).
```
Comment (shaoshuai0102) [inline README.md]: are still working...

File: README.md [hunk 3]
```diff
@@ -43,7 +43,11 @@ Node.js >= 6.0.0 required.
...
+>
+> Welcome to join us to translate egg document, we are glad to see that.
```
Comment (shaoshuai0102) [inline README.md]: documents

File: README.md (hunk 1 above)
Comment (popomore) [inline README.md]: resources?

**Comment by popomore** [comment]
in one line?

File: README.md (hunk 2 above)
Comment (popomore) [inline README.md]: The document is still [working in process](https://github.com/eggjs/egg/issues/363),

**Comment by atian25** [comment]
updated

File: README.md [hunk 4]
```diff
@@ -43,7 +43,8 @@ Node.js >= 6.0.0 required.
 - [Plugins](https://github.com/search?q=topic%3Aegg-plugin&type=Repositories)
 - [Frameworks](https://github.com/search?q=topic%3Aegg-framework&type=Repositories)
 
-> English docs is still working in process, see [#360](https://github.com/eggjs/egg/issues/360).
+> All the sources, comments and the README is in English, only the English guide documents are still [working in process](https://github.com/eggjs/egg/issues/363).
```
Comment (popomore) [inline README.md]: sources 这里是指源文件？

Comment (atian25) [inline README.md]: 嗯, 干掉逗号?

Comment (popomore) [inline README.md]: source code?

Comment (atian25) [inline README.md]: 应该想表达代码注释都是英文的, 源码肯定是英文的, 哈哈

Comment (atian25) [inline README.md]: `All the jsdoc, comments and the README` 怎么样?

Comment (shaoshuai0102) [inline README.md]: All the resources are in English(comments, README, etc.), with one exception, guides are still [working in process](https://github.com/eggjs/egg/issues/363).


别提 source code 了，难道代码还有不是英文的。。哈哈

Comment (atian25) [inline README.md]: 信不信我今晚提交个易语言上来

Comment (atian25) [inline README.md]: 改了

**Pull request merged**